sdk/forms/src/main/java/com/klaviyo/forms/BridgeMessage.kt

```diff
@@ -0,0 +1,163 @@
+package com.klaviyo.forms
+
+import com.klaviyo.analytics.model.Event
+import com.klaviyo.analytics.model.EventKey
+import com.klaviyo.analytics.networking.requests.AggregateEventPayload
+import com.klaviyo.core.Registry
+import java.io.Serializable
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * This should be updated with any new message types we add coming from the onsite-in-app-forms
+ */
+internal sealed class BridgeMessage {
+
+    data object HandShook : BridgeMessage()
+
+    data class Show(
+        val formId: String
+    ) : BridgeMessage()
+
+    data class AggregateEventTracked(
+        val payload: AggregateEventPayload
+    ) : BridgeMessage()
+
+    data class ProfileEvent(
+        val event: Event
+    ) : BridgeMessage()
+
+    data class DeepLink(
+        val route: String
+    ) : BridgeMessage()
+
+    data class Close(
+        val formId: String
+    ) : BridgeMessage()
+
+    data class Abort(
+        val reason: String
+    ) : BridgeMessage()
+
+    companion object {
+        private const val IAF_MESSAGE_DATA_KEY = "data"
```

Comment:
I moved everything out of `Constants` file and eliminated single-use constants. Generally best to avoid constants files -- [reference for anyone interested](https://www.hacklewayne.com/the-constants-file-is-an-anti-pattern-so-is-the-interface-folder-placement-by-functionality-not-technical-concerns)